### ISSUE TITLE
apiDispatcher is not defined

### ISSUE THREAD

**Issue by andyhmltn** [spolu/breach_core#87]
Hey,

I've installed all the dependencies via NPM but I'm getting this error:

```
ReferenceError: apiDispatcher is not defined 
    at Object.<anonymous> (/apps/breach/node_modules/exo_browser/lib/api.js:22:20)
```

Is there anything else to do? Running `Ubuntu 12.04` with `3.8.0-42-generic`

**Comment by jsrn** [comment]
I had that problem when I was trying to run the browser straight out of the git repository (i.e `$ node index.js`. I had better luck running the shellscript packaged with the download on the main breach.cc page.

However, I had to include the chrome sandbox binary in my path to get it to run.

``` bash
$ CHROME_DEVEL_SANDBOX=/opt/google/chrome/chrome-sandbox ./breach
```

I hope that helps.

**Comment by andyhmltn** [comment]
Hmm, interesting. The reason why I tried to run it like the above was because the executable from the website didn't work. I get this error using the above:

`CHROME_DEVEL_SANDBOX=/opt/google/chrome/chrome-sandbox ./breach
/apps/breach/breach-v0.3.20-alpha.5-linux-x64/__AUTO_UPDATE_BUNDLE__/exo_browser/exo_browser: error while loading share
d libraries: libharfbuzz.so.0: cannot open shared object file: No such file or directory`

**Comment by jsrn** [comment]
Looks like your problem might be related to #92. Hopefully `apt-get install libharfbuzz0` will be the end of your problems.

**Comment by andyhmltn** [comment]
I tried that, and after downloading the deb because apt couldn't find the package, I still get the same error :crying_cat_face:

**Comment by jsrn** [comment]
Have you tried coping libharfbuzz.so.0 into `__AUTO_UPDATE_BUNDLE__/exo_browser/lib`? Seemed to help someone from another ticket. Sadly I'm at work, so I can't really investigate further. Good luck!

**Comment by andyhmltn** [comment]
Well, that fixed that error horray! Now I get this: `undefined symbol: g_type_class_adjust_private_offset`

**Comment by spolu** [comment]
This should be solved by the release of v0.3.22-alpha.6
Linux x64: http://bit.ly/1p8Jkmc

Make sure to check README in tarball or https://github.com/breach/breach_core/wiki/Running-Breach-on-Linux
